extensions/amp-facebook-comments/amp-facebook-comments.md

```diff
@@ -62,7 +62,7 @@ You can use the `amp-facebook-comments` component to embed the [Facebook comment
   </tr>
   <tr>
     <td width="40%"><strong>data-colorscheme (optional)</strong></td>
-    <td>The color scheme. For details, see the <a href="https://developers.facebook.com/docs/plugins/comments">Facebook comments documentation</a>.</td>
+    <td>The color scheme. For details, see the <a href="https://developers.facebook.com/docs/plugins/comments">Facebook comments documentation</a>. <br><br><em>Note: This currently may not work due to a bug in Facebook's SDK.</em></td>
```

Comment:
Link to a bug on our end tracking this. 

Link to a bug that we have filed on facebook?

Reply:
@nainar To track this on our end, do I just file an issue?

Reply:
Yup just file an issue. 

Facebook's response is disappointing here but yes let's link to the bug here as well.

Reply:
Added both.